### PR TITLE
Actually store registered etcd listeners.

### LINKED ISSUE #460: Etcd is not properly stopped.
When stopping etcd, it tries to close all client listeners: https://github.com/kubernetes/minikube/blob/32e4c9d26b761231e986a374c66e568b18e3391f/pkg/localkube/etcd.go#L122.

However, the client listeners are not set when starting etcd: https://github.com/kubernetes/minikube/blob/32e4c9d26b761231e986a374c66e568b18e3391f/pkg/localkube/etcd.go#L98

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes #460

### CODE REVIEW

File: pkg/localkube/etcd.go [hunk 1]
```diff
@@ -95,14 +95,14 @@ func (e *EtcdServer) Start() {
 	}
 
 	// create client listeners
-	clientListeners := createListenersOrPanic(e.config.ClientURLs)
+	e.clientListens = createListenersOrPanic(e.config.ClientURLs)
```
Comment (jimmidyson) [inline pkg/localkube/etcd.go]: Shall we rename this field to `clientListeners`?

Comment (dlorenc) [inline pkg/localkube/etcd.go]: Done. Autocomplete was so good I didn't even notice it was different :)

**Comment by jimmidyson** [comment]
LGTM

**Comment by jimmidyson** [comment]
Thanks!

**Pull request merged**